vulnerabilities/precision-loss.md

```diff
@@ -0,0 +1,39 @@
+## Precision Loss
+
+Solidity uses fixed point arithmetic, that means it doesn't support decimal value. As a result, any decimal value is truncated. Thus, performing numerical operations can cause precision loss especially when division is performed before multiplication.
+
+```solidity
+uint a = 11;
+uint b = 2;
+uint c = 10;
+
+function funcA() public {
+    return a / b * c; 
+}
+
+function funcB() public {
+    return a * c / b;
+}
+```
+
+In the above code example, ``funcA()`` returns:
+```soldiity
+11 / 2 * 10 = 50
+```
+and ``funcB()`` returns:
+```solidity
+11 * 2 / 10 = 55
```

Comment:
This should actually return 2. I think your example is mixed up

Reply:
Yeah, Fixed in the latest commit!